cluster-autoscaler/simulator/framework/handle.go

```diff
@@ -36,27 +37,36 @@ type Handle struct {
 }
 
 // NewHandle builds a framework Handle based on the provided informers and scheduler config.
-func NewHandle(informerFactory informers.SharedInformerFactory, schedConfig *schedulerconfig.KubeSchedulerConfiguration) (*Handle, error) {
+func NewHandle(informerFactory informers.SharedInformerFactory, schedConfig *schedulerconfig.KubeSchedulerConfiguration, draEnabled bool) (*Handle, error) {
 	if schedConfig == nil {
 		var err error
 		schedConfig, err = schedulerconfiglatest.Default()
 		if err != nil {
 			return nil, fmt.Errorf("couldn't create scheduler config: %v", err)
 		}
 	}
-
 	if len(schedConfig.Profiles) != 1 {
 		return nil, fmt.Errorf("unexpected scheduler config: expected one scheduler profile only (found %d profiles)", len(schedConfig.Profiles))
 	}
+	schedProfile := &schedConfig.Profiles[0]
+
 	sharedLister := NewDelegatingSchedulerSharedLister()
+	opts := []schedulerframeworkruntime.Option{
+		schedulerframeworkruntime.WithInformerFactory(informerFactory),
+		schedulerframeworkruntime.WithSnapshotSharedLister(sharedLister),
+	}
+
+	if draEnabled {
```

Comment:
nit: It would be a bit more readable to place `schedProfile :=` before `if draEnabled`.